man/systemd-analyze.xml

```diff
@@ -104,6 +104,12 @@
     <cmdsynopsis>
       <command>systemd-analyze</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">seccomp-filter</arg>
```

Comment:
looks great conceptually, but I think we should call this "syscall-filter" instead. I'd use "seccomp" as the term for the technology behind a veriety of sandboxing options, among them SystemCallFilter=, RestrictAddressFamilies=, RestrictRealtime= and so on. But the syscall groups only apply to one specific option which is SystemCallFilter=. Hence I think we should be specific here, and clarify the relationship of this new systemd-analyze option with the actual unit file option and call the verb "syscall-filter" (or: "system-call-filter", but I figure having different brevity rules for unit file names and command line verbs is OK)